zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -146,32 +146,34 @@ public void testRemoveButton() throws InterruptedException {
       createNewNote();
 
       waitForParagraph(1, "READY");
-      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
-      driver.findElement(By.xpath(getParagraphXPath(1) + "//ul/li/a[@ng-click='insertNew()']")).click();
-      waitForParagraph(2, "READY");
-      Integer oldNosOfParas = driver.findElements(By.xpath("//div[@ng-controller=\"ParagraphCtrl\"]")).size();
-      collector.checkThat("Before Remove : Number of paragraphs are ",
-              oldNosOfParas,
-              CoreMatchers.equalTo(2));
-      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
-      driver.findElement(By.xpath(getParagraphXPath(1) + "//ul/li/a[@ng-click='removeParagraph()']")).click();
-      sleep(1000, true);
-      driver.findElement(By.xpath("//div[@class='modal-dialog'][contains(.,'delete this paragraph')]" +
-              "//div[@class='modal-footer']//button[contains(.,'OK')]")).click();
-      Integer newNosOfParas = driver.findElements(By.xpath("//div[@ng-controller=\"ParagraphCtrl\"]")).size();
-      collector.checkThat("After Remove : Number of paragraphs are",
-              oldNosOfParas-1,
-              CoreMatchers.equalTo(newNosOfParas));
+
+      collector.checkThat("Default Width is 12 ",
+              driver.findElement(By.xpath("//div[contains(@class,'col-md-12')]")).isDisplayed(),
+              CoreMatchers.equalTo(true));
+      for (Integer newWidth = 1; newWidth <= 11; newWidth++) {
+        driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
+        String visibleText = newWidth.toString();
+        new Select(driver.findElement(By.xpath(getParagraphXPath(1)
+                + "//ul/li/a/form/select[(@ng-change='changeColWidth()')]"))).selectByVisibleText(visibleText);
+        collector.checkThat("New Width is : " + newWidth,
+                driver.findElement(By.xpath("//div[contains(@class,'col-md-" + newWidth + "')]")).isDisplayed(),
+                CoreMatchers.equalTo(true));
+      }
+
+
       ZeppelinITUtils.sleep(1000, false);
+
       deleteTestNotebook(driver);
 
     } catch (Exception e) {
-        LOG.error("Exception in ParagraphActionsIT while testRemoveButton ", e);
-        File scrFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
-        throw e;
+      LOG.error("Exception in ParagraphActionsIT while testWidth ", e);
+      File scrFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
+      throw e;
```

Comment:
Also, since all other selenium tests are using "handleException" can you use that here as well ?